Some minor additions

temp1011 opened the pull request:
Resolves https://github.com/ForestryMC/ForestryMC/issues/1923

If you think some hives should not be flammable (maybe end?) or the burn values are wrong (I used values for logs) please comment below.

Hope this helps

temp1011 committed:
make non hellish hives flammable

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ForestryMC/ForestryMC?pullRequest=1930) <br/>All committers have signed the CLA.

temp1011 committed:
update config in apiculture

temp1011 committed:
add damage option to hives based on config

temp1011 commented:
Now resolves https://github.com/ForestryMC/ForestryMC/issues/1710 (and removes a todo I came across). I know these are weird issues to do but I'm not good enough at coding to be able to fix some of the more complicated bugs so I'm trying to help where I can. If the min and max values for ticksPerBeeWorkCycle are wrong please comment.

temp1011 committed:
Woops

Nedelosk commented on src/main/java/forestry/apiculture/worldgen/HiveDescription.java:
```diff
@@ -146,6 +146,10 @@ public float getGenChance() {
 	public void postGen(World world, Random rand, BlockPos pos) {
+		if(isGoodBiome(world.getBiome(pos)))
+		{
+			Blocks.FIRE.setFireInfo(world.getBlockState(pos).getBlock(), 5, 5);
```
This does not work like you think. `setFireInfo` sets the fire info for all forestry hives because all forestry hives have the same block instance. You have to implement the methods`getFlammability`, `getFireSpreadSpeed` and `isFlammable` in the class of the hive block.

temp1011 replied:
Ah OK. Makes sense.

temp1011 committed:
Fix hive flammability

Hive placed in the overworld burned whereas hive placed in nether did not in my tests. Hopefully this works better.

temp1011 commented:
Hopefully this should be the correct implementation now.

temp1011 committed:
Merge branch 'mc-1.12' into mc-1.12

temp1011 committed:
Merge branch 'mc-1.12' into mc-1.12

Nedelosk commented on src/main/java/forestry/apiculture/blocks/BlockBeeHives.java:
```diff
@@ -197,5 +199,30 @@ public EnumBlockRenderType getRenderType(IBlockState state) {
+	@Override
+	public int getFireSpreadSpeed(IBlockAccess world, BlockPos pos, EnumFacing face)
+	{
+		if(!BiomeHelper.isBiomeHellish(world.getBiome(pos)))
```
You don't have to test if the hive is in the nether, because the nether hive is from binnie and not from forestry.

temp1011 replied:
ah, I didn't realise binnie had their own hive material and block

temp1011 committed:
No biome check
